test/bin/mdx-rule/misc/duniverse-mode/dune.gen.expected

```diff
@@ -9,6 +9,11 @@
   prelude.ml)
  (action
   (progn
-   (run ocaml-mdx test --prelude=prelude.ml --prelude-str
-     "#require \"prelude-str\"" %{x})
+   (run
```

Comment:
I have a couple remarks about that:
- it seems wrong that formatting depends on the version of dune rather of the dune-lang. I thought it was now a pure function of dune-lang
- if there's an easy way to use (enabled_if) to gate that formatting on the dune version, that would be a bit better I think
- if this is a test for mdx rule, it'd be good to add a note somewhere in mdx rule of the things that can be undone once this feature goes away

Reply:
I agree about the formatting issues but as a MDX maintainer I have no control over this and it turns out it does depend on the dune version, not the language version.

Using enabled_if means I need to duplicate the test which frankly feels like unnecessary extra work. mdx-rule is on its way out so we're preserving the test but that's pretty much it.

One alternative is to simply require dune >= 2.8 as this formatting issue might show up for our own dune files, that way we shield ourselves from having to deal with this in the future and having to revert the changes in this PR.

Reply:
I think it depending on the dune version and not dune-lang would be a dune bug. I'll ask the maintainers.

Reply:
It would be a bug indeed, but in a small experiment I cannot reproduce the problem locally. Rerproducing the issue in a small sample would be helpful.